### PR TITLE
Improve labelling of vertical lines in plots

### PR DESCRIPTION
**Pull request by RosalynLP**
- Created function called `add_lines_and_notes` which allows you to add multiple lines and corresponding notes to a plot, rather than repeating code each time
- Applied this to existing charts with multiple lines
- Notes added in the legend to avoid overlap with data

### CODE REVIEW

File: shiny_app/functions/functions_server.R [hunk 1]
```diff
@@ -1568,11 +1536,13 @@ plot_VaccineWastage <- function(dataset, area = T) {
                       xref = "paper",
                       y = 0.7,
                       x = frac2,
-                      text = "<b>All Doses</b>",
+                      text = "All Doses",
```
Comment (RosalynLP) [inline shiny_app/functions/functions_server.R:1539]:
```suggestion
                      text = "<b>All Doses</b>",
```

File: shiny_app/functions/functions_server.R [hunk 2]
```diff
@@ -1559,7 +1527,7 @@ plot_VaccineWastage <- function(dataset, area = T) {
                      xref = "paper",
                      y = 0.7,
                      x = frac1,
-                     text = "<b>Doses 1 & 2</b>",
+                     text = "Doses 1 & 2",
```
Comment (RosalynLP) [inline shiny_app/functions/functions_server.R:1530]:
```suggestion
                     text = "<b>Doses 1 & 2</b>",
```

**Review by RosalynLP — COMMENTED** [review]
Doing a self review here, can probs add these back in :)

**Comment by RosalynLP** [comment]
@rubenv95 this is ready to review now 😅

File: shiny_app/functions/functions_server.R [hunk 3]
```diff
@@ -17,44 +17,41 @@ vline = function(x, width=3.0, color="black", ...) {
   return(l_shape)
 }
 
-annotation = function(frac, y, note, ax, ay, color){
-  ann = list(yref = "paper",
+annotation <- function(frac, y, note, color){
+  ann <- list(yref = "paper",
```
Comment (rubenv95) [inline shiny_app/functions/functions_server.R:21]: Boooo, I love =

Comment (RosalynLP) [reply shiny_app/functions/functions_server.R:21]: I know but just trying to keep styling consistent

**Review by rubenv95 — APPROVED** [review]
Looks good - makes sense to add it to the legend, makes it more clear. 
Can you just remove or name the legend for proportion chart trace 0 before merging? 
image

**Pull request merged**